internal/pkg/api/handleCheckin.go

```diff
@@ -911,6 +918,47 @@ func processPolicy(ctx context.Context, zlog zerolog.Logger, bulker bulk.Bulk, a
 	return &resp, nil
 }
 
+func prepareOTelExporters(outputs map[string]map[string]interface{}, exporters map[string]any) error {
```

Comment:
```suggestion
func prepareOTelExporters(outputs map[string]map[string]any, exporters map[string]any) error {
```
(nitpick) let's try to be consistent for a func

Reply:
also please add a comment stating what this func does